modules/core/client/directives/page-title.client.directive.js

```diff
@@ -19,13 +19,17 @@
 
       function listener(event, toState) {
         var applicationCoreTitle = 'MEAN.js',
-          separeteBy = ' - ';
+          separeteBy = ' - ',
```

Comment:
`separator` is probably more concise.

@PierreBrisorgueil I actually meant `separateBy`, sorry I wasn't clear on that. However, I do prefer Mikael's suggestion.